play/src/components/ResultSelector.tsx

```diff
@@ -105,7 +107,7 @@ export default function ResultSelector({
         </Typography>
       </Box>
       <Box style={{ display: 'flex', alignItems: 'center' }}>
-        {currentState === GameSessionState.PHASE_2_RESULTS && ( // if in phase 2, display percentage text
+        {currentState === GameSessionState.PHASE_2_DISCUSS && ( // changed this
```

Comment:
can probably delete the comment, as it doesn't describe the changes etc

Reply:
gone